docs/gpa.md

```diff
@@ -14,6 +14,28 @@ This document annotates selected parts of the GPA, with details on how to publis
 
    References from Articles VII:4, VII:5, IX:8, IX:9, X:7, X:9, X:11, XIII:2, XIV:1, XVI:2 are not yet added.
 ```
+<div markdown=1>
+
+# Common operations
```

Comment:
Increase heading levels. Only the page title should be heading 1.

Reply:
Or, we might want to move this to a separate page, because it's quite technical, and not the first thing we want to show people who are looking for a mapping between OCDS and the GPA.

Reply:
Done inscreasing heading level